### PR TITLE
Add Router, errorAtom, renderLogic

### PR DESCRIPTION
**Pull request by toddkao**

### CODE REVIEW

File: packages/widget-v2/tsconfig.app.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     /* Aliases */
     "baseUrl": ".",
     "paths": {
-      "@*": ["src/*"]
+      "@/*": ["src/*"]
```
Comment (toddkao) [inline packages/widget-v2/tsconfig.app.json:29]: This fixes auto imports / ide from suggesting `@components` instead of `@/components` which isn't valid with this config

File: packages/widget-v2/src/widget/Router.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        </ErrorBoundary>
+      );
+    case Routes.SwapExecutionPage:
+      return (
+        <ErrorBoundary fallback={null} onError={(error) => setError(error)}>
+          <SwapExecutionPage />
+        </ErrorBoundary>
+      );
+  }
+};
```
Comment (toddkao) [inline packages/widget-v2/src/widget/Router.tsx:31]: I tested manually throwing an error within SwapPage and it successfully sets errorAtom state and renders ErrorPage

Comment (toddkao) [reply packages/widget-v2/src/widget/Router.tsx:31]: I also tested and can confirm that these error boundaries **will not** catch any errors thrown within a modal, we will need to handles those seperately

File: packages/widget-v2/src/pages/ErrorPage/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+export const ErrorPage = () => {
+  const resetError = useResetAtom(errorAtom);
+
+  return (
+    <div>
+      error page
+      <button onClick={() => resetError()}>reset</button>
+    </div>
+  );
+};
```
Comment (toddkao) [inline packages/widget-v2/src/pages/ErrorPage/ErrorPage.tsx:13]: this is temp, i created a ticket to actually create the ErrorPage properly

File: packages/widget-v2/src/stories/SwapExecutionPage.stories.tsx [hunk 4]
```diff
@@ -57,7 +56,5 @@ type Story = StoryObj<typeof meta>;
 export default meta;
 
 export const SwapExecutionPageExample: Story = {
-  args: {
-    operations: operations as Operation[],
-  },
+  args: {},
```
Comment (codingki) [inline packages/widget-v2/src/stories/SwapExecutionPage.stories.tsx:59]: any reason why you move it inside?

**Review by codingki — APPROVED** [review]
lgtm

File: packages/widget-v2/src/stories/SwapExecutionPage.stories.tsx (hunk 4 above)
Comment (toddkao) [reply packages/widget-v2/src/stories/SwapExecutionPage.stories.tsx:59]: @codingki yeah I figure that eventually this data will exist in an atom, so we won't be passing it into the component as a prop, so I decided to just move it inside for now (the usage of the component is more similar to how it'll be when we have the data in an atom)
